app/src/main/res/values-hi/strings.xml

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="about">बारे में</string>
+    <string name="action_settings">सेटिंग्स</string>
+    <string name="cantDownload">डाउनलोड नहीं कर सकते</string>
+    <string name="default_mode_notification">10 मिनट पहले</string>
+    <string name="default_mode_api">डेवलपर</string>
+    <string name="hour_before"> 1 घंटा</string>
+    <string name="download_failed"> डाउनलोड विफल</string>
+    <string name="close_menu">मेनू को बंद करें</string>
+    <string name="menu_bookmarks"> बुकमार्क</string>
+    <string name="main_menu">मुख्य मेनू</string>
+    <string name="menu_locations">स्थान</string>
+    <string name="menu_map">नक्शा</string>
+    <string name="menu_speakers">वक्ताओं</string>
+    <string name="menu_sponsor">प्रायोजक</string>
+    <string name="menu_tracks">पटरियों</string>
+    <string name="refresh_failed">ताज़ा नहीं कर सका</string>
+    <string name="action_example">उदाहरण कार्रवाई</string>
+    <string name="download_complete">डाउनलोड हो गया</string>
+    <string name="fb"></string>
+    <string name="settings_mode_api">एपीआई मोड</string>
+    <string name="settings_mode_notification"> अधिसूचना मोड</string>
+    <string name="share">शेयर</string>
+    <string name="share_links">शेयरिंग यूआरएल</string>
+    <string name="splash">छप</string>
```

Comment:
Please son't take the literal meaning :P. I think you should just write splash in hindi here.